recipes/pytorchinfo/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pytorchinfo" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/53/d9/2b811d1c0812e9ef23e6cf2dbe022becbe6c5ab065e33fd80ee05c0cd996/torchinfo-1.8.0.tar.gz
+  # https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchinfo-{{ version }}.tar.gz
+  sha256: 72e94b0e9a3e64dc583a8e5b7940b8938a1ac0f033f795457f27e6f4e7afa2e9
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or osx]
```

```suggestion
  noarch: python
```

while we don't have pytorch on windows, this would enable users to install it on mac without massive rebuilds from your part.

windows users will need to find a way to build a compatible windows version of pytorch themselves.